src/main/java/frc/robot/Robot.java

```diff
@@ -32,8 +36,21 @@ public void robotInit() {
         coralElevator = new CoralElevator();
         coralGripper = new CoralGripper();
         coralArm = new CoralArm();
+        xbox = new XboxController(0);
+        ph = new PneumaticHub(1);
+        if (ph.getModuleNumber() != 1) {
+            throw new RuntimeException("Pneumatic Hub not detected!");
+        }
+        cpr = new Compressor(1, PneumaticsModuleType.REVPH);
+        cpr.enableAnalog(minPresseure, maxPresseure);
+        System.out.println("PH Module ID: " + ph.getModuleNumber());
+        ph.enableCompressorAnalog(minPresseure,maxPresseure);
+        ph.enableCompressorDigital();
```

Comment:
you can do either digital, analog or hybrid. But not digital, analog. As far as I can tell, this will simply be digital mode as it is the last called.